onko_dicom.py

```diff
@@ -67,6 +73,24 @@ def create_menu(self):
 
         logger.info("Initialised Menu within OnkoDicom")
 
+    def err_msg(self, title, msg):
+        """Error handling messagebox"""
+        msg_box = QMessageBox(self, title, msg)
+        msg_box.setIcon(QMessageBox.Critical)
+        msg_box.setWindowTitle(title)
+        msg_box.setText(msg)
+        msg_box.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+        button_y = msg_box.button(QMessageBox.Yes)
+        button_y.setText('Force Open')
+        button_n = msg_box.button(QMessageBox.No)
+        button_n.setText('Abort')
+        msg_box.exec()
+
+        if msg_box.clickedButton() == button_y:
+            return 'F'
+        elif msg_box.clickedButton() == button_n:
+            return 'A'
```

Comment:
I think this should return true/false, any reason your using F and A

Reply:
Not, these are only for error handling button response 
if response == ‘F’:
     return true